Jenkinsfile-Release

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+@Library(['private-pipeline-library', 'jenkins-shared', 'nxrm-jenkins-shared']) _
+
+final jira = [
+    versionPrefix: '', project: 'NEXUS', projectId: '12410',
+    credentialId : 'jenkins-jira', autoRelease: true, failOnError: true
+]
+
+final jiraVersionMappings = [
+  'nexus-repository-manager-single-instance': 'helm-nxrm',
+  'nexus-repository-manager-aws-resilient-single-instance': 'helm-nxrm-aws-resiliency'
+]
+
+final chartLocation = [
+  'nexus-repository-manager-single-instance': 'single-inst-oss-pro-kubernetes',
+  'nexus-repository-manager-aws-resilient-single-instance': 'aws-single-instance-resiliency'
+]
+
+properties([
+  parameters([
+    choice(
+      choices: ['', 'nexus-repository-manager-single-instance', 'nexus-repository-manager-aws-resilient-single-instance'],
+      name: 'chart',
+      description: 'Chart to deploy.',
```

Comment:
_Is_ there a use-case where you'd be updating versions on one chart but not the other?  You might want to lose this parameter and always update both charts from one invocation of the build. The old build had this parameter because the 2 products (nxrm and iq) had different releases.

Reply:
Done: https://github.com/sonatype/nxrm3-helm-repository/pull/9/commits/2aa0f5dbf98e0757c61bda586bf64dbe6a0c570b